Tests/demistomock/demistomock.py

```diff
@@ -479,59 +640,99 @@ def executeCommand(command, args):
 
 
 def getParam(param):
+    """(Integration only)
+    Extracts given parameter from the integration parameters object
+
+    :type param: ``str``
+    :param param: Integration parameter to get value of
+
+    :return: Integration parameter value
+    :rtype: ``str``
+    """
     return params().get(param)
 
 
 def getArg(arg):
+    """
+    Extracts given argument from the arguments object
+
+    :type param: ``str``
+    :param param: Argument to get value of
+
+    :return: Argument value
+    :rtype: ``str``
+    """
     return args().get(arg)
 
 
 def setIntegrationContext(context):
+    """(Integration only)
+    Stores given object in the IntegrationContext object
+
+    :type obj: ``dict``
+    :param obj: The object to store
+
+    :return: No data returned
+    :rtype: ``None``
+    """
     global integrationContext
     integrationContext = context
 
 
 def getIntegrationContext():
+    """(Integration only)
+    Retrieves the IntegrationContext object
+
+    :return: IntegrationContext object
+    :rtype: ``dict``
+    """
     return integrationContext
 
 
 def setIntegrationContextVersioned(context, version=-1, sync=False):
-    """
-    Set the integration context with a version.
-    Args:
-        context: The context to set.
-        version: The context version to set. If the version is older than the current, an error will be thrown.
-        sync: Whether to save the context to the DB right away. If false,
-         the context will be saved at the end of the command.
+    """(Integration only)
+    Stores given object in the IntegrationContext object in given version
 
+    :type context: ``dict``
+    :param context: The object to store
+
+    :type version: ``int``
+    :param version: The context version to set. If the version is older than the current, an error will be thrown.
+
+    :type sync: ``bool``
+    :param sync: Whether to save the context to the DB right away.
+    If false, the context will be saved at the end of the command.
+
+    :return: No data returned
+    :rtype: ``None``
     """
     global integrationContext
     integrationContext = context
 
 
 def getIntegrationContextVersioned(refresh=False):
-    """
-    Get the integration context with a version
-    Args:
-        refresh: Whether to get the integration context straight from the DB and not from the instance memory.
+    """(Integration only)
+    Retrieves the versioned IntegrationContext object
+
+    :type refresh: ``bool``
+    :param refresh: Whether to get the integration context straight from the DB and not from the instance memory.
 
-    Returns:
-        Dict with a key of context and a key of version.
+    :return: IntegrationContext versioned object
+    :rtype: ``dict``
     """
     return integrationContext
 
 
 def incidents(incidents=None):
-    """
-    In Scripts this returns the `Incidents` list from the context
-
-    In integrations this is used to return incidents to the server
+    """(Integration only)
```

Comment:
Not accurate. This function also exists in scripts. In scripts you get the incidents list from the context.